app/charts/index.spec.ts

```diff
@@ -69,12 +69,7 @@ describe("initial config", () => {
   it("should create an initial table config with column order based on dimension order", () => {
     const config = getInitialConfig({
       chartType: "table",
-      iris: [
-        {
-          iri: "https://environment.ld.admin.ch/foen/nfi",
-          publishIri: "https://environment.ld.admin.ch/foen/nfi",
-        },
-      ],
+      iris: [{ iri: "https://environment.ld.admin.ch/foen/nfi" }],
```

Comment:
If I understand well, the componentIds will already include the "publishIri" (I am still fuzzy on why we need publish iris 🙈 ) ?

Reply:
Yeah, I think it was added just in case – what happens now is that when we load a config, we upgrade the cube iri to latest one and overwrite it on the client-side. So `publishIri` is always stored in a db, and we know it before it gets upgraded, so I removed the `publishIri` 👍

Reply:
We could need them in case there's a feature to "pin" cube version and disable auto-upgrade behavior, but it's not something that's coming soon (or ever?) :)

Reply:
But then since we already have the non upgrade cube iri, it would not be an additional field, but more of a boolean "doNotUpgrade" right ?

Reply:
Exactly, I think this would be more clear 👍